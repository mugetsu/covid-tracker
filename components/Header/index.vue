<template>
  <header>
  </header>
</template>

<script>
export default {
  name: 'Header',
  data() {
    return {}
  }
}
</script>

<style lang="scss" scoped>
// header {
//   padding-top: 32px;
//   padding-bottom: 32px;
//   font-size: 1.8rem;
//   letter-spacing: 1px;
//   font-weight: 100;
// }
</style>
