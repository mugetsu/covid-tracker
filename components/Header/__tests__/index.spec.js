import { shallowMount } from '@vue/test-utils'
import Header from '../index.vue'

const factory = () => {
  return shallowMount(Header, {})
}

describe('Header', () => {
  test('mounts properly', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
