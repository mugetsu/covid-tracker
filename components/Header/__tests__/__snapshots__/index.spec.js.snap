// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `<header></header>`;
